cinch/roles/jenkins_master/templates/advised_security.groovy

```diff
@@ -0,0 +1,24 @@
+import jenkins.model.*
```

Comment:
Please add semicolons to the end of statements throughout this file.

Reply:
Sure.